setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 110
+ignore = E133, E226, E228, E251, N802, N803, W391
```

Comment:
I will see if I can have a global file for it. It was the only way I knew of to ensure that people get the right answer running `flake8` from the command line.